content/docs/command-reference/add.md

```diff
@@ -16,23 +16,32 @@ positional arguments:
 ## Description
 
 The `dvc add` command is analogous to `git add`, in that it makes DVC aware of
-the target data, as a first step to version it. It creates a
+the target data, as a simple step to version it. It creates a
 [DVC-file](/doc/user-guide/dvc-file-format) to track the added data.
 
-The `targets` are files or directories to add with this command, that are turned
-into <abbr>data artifacts</abbr> of the <abbr>project</abbr>. By default, these
-are committed to the <abbr>cache</abbr> (use the `--no-commit` option to avoid
-this, and `dvc commit` to finish the process when needed).
+The `targets` are files or directories to add, which are turned into <abbr>data
+artifacts</abbr> of the <abbr>project</abbr>. By default, these are stored in
+the <abbr>cache</abbr> (use the `--no-commit` option to avoid this, and
+`dvc commit` to finish the process when needed).
 
-Note that [external data](/doc/user-guide/managing-external-data) (targets
+This command can be used to
+[version control](/doc/use-cases/versioning-data-and-model-files) any large file
+or directory that's too big for Git. Usually, these comprise the initial data
```

Comment:
any large files, models, directories.

Let's remove this "usually, ..." part. I still don't like this reference to pipelines. Note below is enough.

Also, I would remove Not that external data is supported. At least I don't like again the way we put it.

In general, my concern that we hide the simple case behind many reference to different cases and features. And we don't even explain them well. All this stuff together is hard to read and understand and leads to endless stream of misunderstadings on Discrod. It would be way better to write a paragraph to explain the default phylosophy - Git like in a sense that data is part of the workspace.

Reply:
All addressed in c51e2ad7.

Please note most of this text was already there (I think the change here originally was just the link), and I definitely agree it can be improved. I'll keep in mind this motif:

> way better to write a paragraph to explain the default philosophy